README.md

```diff
@@ -61,6 +61,13 @@ options object.
   sampleRate <= 99```
 * ```clientId: null```<br/>
   An identifier used to disambiguate this client.
+* ```ca: <same as node.js tls.connect() description>```  
```

Comment:
Do we need to provide these options inconjunction? Might to specify that in the README.